src/pages/Transaction/helpers.tsx

```diff
@@ -48,6 +48,10 @@ export function getLearnMoreTooltip(txnField: string): JSX.Element | null {
           link="https://aptos.dev/en/network/glossary#expiration-time"
         />
       );
+    case "fee_payer":
+      return (
+        <LearnMoreTooltip text="Account that paid for the gas fee of the transaction, separate of the sender." />
```

Comment:
Fee payer can also be sender https://deploy-preview-838--aptos-explorer.netlify.app/txn/1831974613?network=mainnet maybe "separate of the sender" -> "instead of sender"